letsencrypt/data/ir_cron.xml

```diff
@@ -2,12 +2,12 @@
 <openerp>
```

Comment:
This can be changed into odoo as well (and the noupdate made an attribute of the odoo element and the data element removed).